File: backend-node-portifolio/src/api/domain/models/Image.ts
import { Section } from "./Section";

export class Image{
    public Id: number;
    public Guid: string;
    public Sequence: number;
    public UrlImage: string;
    public Caption?: string;
    public SectionId: number;
    public Section: Section;
    constructor() {
        this.Id = 0;
        this.Guid = "";
        this.Sequence = 0;
        this.UrlImage = "";
        this.Caption = "";
        this.SectionId = 0;
        this.Section = new Section();
    }
}

File: backend-portifolio/Models/Image.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace backend_portifolio.Models
{
    public class Image
    {
        public long Id { get; set; }
        public Guid Guid {get; set; }
        public int Sequence {get; set;}
        public string UrlImage { get; set; }
        public string Caption {get; set;}
    }
}